glom/test/test_basic.py

```diff
@@ -4,14 +4,13 @@
 
 import pytest
 
-from glom import glom, SKIP, STOP, Path, Inspect, Coalesce, CoalesceError, Val, Call, T, S, Invoke, Spec, Ref
-from glom import Auto, Fill, Iter, A, Vars, Val, Literal, GlomError, Pipe
+from glom import glom, SKIP, STOP, Inspect, Coalesce, Val, Call, T, S, Invoke, Spec, Ref
+from glom import Fill, Iter, Literal, Pipe
 
 import glom.core as glom_core
-from glom.core import UP, ROOT, bbformat, bbrepr
-from glom.mutation import PathAssignError
+from glom.core import UP, bbformat, bbrepr
 
-from glom import OMIT, Let, Literal  # backwards compat
+from glom import OMIT  # backwards compat
```

Comment:
same with the backwards compat ones here. they can probably move up close to the other public ones, but keep the comment.